test_othermlpackages/conda-tensorflow.yaml

```diff
@@ -3,7 +3,9 @@ channels:
 - defaults
 - conda-forge
 dependencies:
-- python==3.7
-- pip>=19.1.1
```

Comment:
Why did we remove this lower bound? Seems sensible to have one.

Reply:
Restored. It was just one of the things I tried before I discovered the actual issue with TF